### ISSUE TITLE
Create PDT user manual with examples

### ISSUE THREAD

**Issue by ermo** [Clockmender/Precision-Drawing-Tools#6]
**Is your feature request related to a problem? Please describe.**
PDT is not very discoverable as it stands -- it seems its audience is mostly experienced CAD users.

**Describe the solution you'd like**
It would be nice to have a manual that targets the casual Blender user/artist getting started with precision modeling for e.g. 3D printing projects or the like.

The manual should have plenty of examples and walkthroughs that exercise all areas of PDT's features.

**Comment by Clockmender** [comment]
Agreed, tha manual needs to be produced, it is a large amount of work, so may take some time. What format should we use here?

**Comment by ermo** [comment]
Let's use github wiki Markdown format.

**Comment by Clockmender** [comment]
Can we include pictures? I am not a markdown experienced chap, more to learn!

**Comment by ermo** [comment]
> Can we include pictures? I am not a markdown experienced chap, more to learn!

https://help.github.com/en/articles/about-wikis

https://help.github.com/en/articles/basic-writing-and-formatting-syntax#relative-links

The image files will need to be part of the repository. Then you can reference them with relative links AFAICT.

https://guides.github.com/features/mastering-markdown/#examples

**Comment by Clockmender** [comment]
Project started, long way to go still.

**Comment by Clockmender** [comment]
Initial pages now complete, more worked examples are required and a page on "Draughting Techniques & CAD Topology", I will attend to this after my holiday.

**Comment by ermo** [comment]
@Clockmender How far along is this in your view?

**Comment by raindropsfromsky** [comment]
Hi

I noticed that the [wiki](https://github.com/Clockmender/Precision-Drawing-Tools/wiki/)  is outdated, and therefore it would confuse the readers. I wanted to open an issue about it, but then I came across this issue.

I think I can help here. 

I have written 10+ whole manuals for different software. The most notable among them is the [User Manual for LMMS](https://lmms.io/pdf/lmms-manual-0.4.12.pdf) and the [Wiki for ReNamer](https://www.den4b.com/wiki/ReNamer). 

I try to figure out the software myself, rather than overly loading the author. Just an introductory chat/discussion would suffice. The author does not have to worry about grammar and formatting, either.

Please let me know if you are interested. We can start immediately and finish the Wiki in 3-4 days.

**Comment by Clockmender** [comment]
I certainly am, under what basis would this be and how much more information would you need.

Since the WiKi was started, we have modified the software somewhat in the current release, 1.4.0.

Let me know what you thoughts are please.

Cheers, Clock.

**Comment by raindropsfromsky** [comment]
Can we chat on gmail or any other platform? You have my gmail ID but I cannot see yours.
Let us sync up directly!

**Comment by Clockmender** [comment]
Ah! I don't use gmail at all, for development I tend to use Discord, my name is:

```CLockmender #4196```

Is that OK for you?

**Comment by raindropsfromsky** [comment]
> `Clockmender #4196`

Sent a friend request on Discord. 
I am in Bangalore, India (IST = UTC+5:30). So we will catch up after 7 hours!

**Comment by Clockmender** [comment]
I am in UK on UTC, so it will be more than 7 hours Ha Ha!

**Comment by Blendersbrontobyte** [comment]
I just Purchased the ReNamer app one of the best I used.  And great help documentation.  PDT This is already a blender addon that comes with blender.  I just recently found out about it.  I am interested in helping with the documentation as well if I could find someone to pay me to do it.   I am a creative person left minded.  The app does a lot of things and it looks like one hell of a lot of work was put into it.  I am having trouble understanding the very first  part.  
![DocumentationExample](https://user-images.githubusercontent.com/59634138/108728088-763de500-74e6-11eb-97d9-da1e025c51bc.png)
Learning and understanding is very hard for me.  But once I understand something I am good at organizing things so they make sense and good at communicating things with word and I have adobe PDF pro and I use word and photoshop daily.  There is a lot going on here with this addon and it must have been a challenge to organize it in the menus.  I never see long tool tips in software but maybe it would be ok here in this complex situation ? tool tips that would help me pin down what does what to what and when and under what circumstances.  I never see long tool tips ( what comes up when I hover )   If given the opportunity and if I could find a sponsor to pay me I would like to help with the documentation as well.

**Comment by Clockmender** [comment]
Hello, sorry for the late reply, I have had some time away from Blender and missed your mail...

There has been a lot of debate about long tool tips and I am on the side of trying to keep them short and succinct, however I am open to suggestions as to how to improve this. Have you read the Wiki yet? there is a lot of info in there and still a way to go to improve it.

https://github.com/Clockmender/Precision-Drawing-Tools/wiki

You might also look at the thread on DevTalk https://devtalk.blender.org/t/making-an-add-on-for-precision-drawing-tools-cad/8637

This shows the development cycle and lots of info on how it is used and what the various sections do and are there for.

On the matter of paying to do the documentation, this is not something I could consider, being retired and having done all the work for free and not charging for the Add-on, but it may be that others might look at this for you.

I am, of course, here to help in any way I can with any questions you might have!

Cheers, Clock.
